.github/actions/create-build/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Create Flowistry build artifacts'
+description: 'Builds Flowistry with release flag'
+inputs:
+  target:
+    required: true
+runs:
+  using: "composite"
+  steps: 
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly-2021-09-23
+        components: rust-src, rustc-dev, llvm-tools-preview
+        target: ${{ inputs.target }}
+        profile: minimal
+        override: true
+    - uses: actions-rs/cargo@v1
+      with:
+        command: build
+        args: --release
```

Comment:
I created this [`composite action`](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) since the `build` job in the `tests` action use the same build steps as the `release` action. If it looks good, it might be good to reuse this in `release.yml` so both actions build the same.